ui/src/EventForm.tsx

```diff
@@ -144,6 +146,8 @@ const EventForm = ({
           isChecked={allDay}
           onChange={(e) => {
             setAllDay(e.target.checked);
```

Comment:
This can be updated so that it only happens in the case that the All Day checkbox is unchecked.

Reply:
Added it in